Library/Homebrew/test/cask/cli/search_spec.rb

```diff
@@ -56,4 +56,16 @@
       Hbc::CLI::Search.run("caskroom")
     }.to output(/^No Cask found for "caskroom"\.\n/).to_stdout
   end
+
+  it "doesn't highlight not installed packages" do
```

Comment:
This should be ”non-installed packages“, which still doesn't sound right, so maybe just ”packages that aren't installed“?